package.json

```diff
@@ -4,15 +4,9 @@
   "description": "Modules to generate test data for commercetools APIs",
   "license": "MIT",
   "private": true,
-  "workspaces": [
-    "core",
-    "models/*"
-  ],
+  "workspaces": ["core", "models"],
   "preconstruct": {
-    "packages": [
-      "core",
-      "models/*"
-    ]
+    "packages": ["core", "models"]
```

Comment:
if we want, `commons` can be its own package on the root folder. it makes more sense.